packages/integrations/react/package.json

```diff
@@ -62,8 +62,8 @@
   "peerDependencies": {
     "@types/react": "^17.0.50 || ^18.0.21",
```

Comment:
Should we add the `@types/react` preview package as well? They are listed as `npm:types-react@beta` and `npm:types-react-dom@beta`. Not sure if the `npm:` prefix works with peer deps though

Reply:
I don't know how to semver this, is it:

```
"@types/react": "^17.0.50 || ^18.0.21 || npm:types-react@beta"
```

I admit this is outside of my wheelhouse

Reply:
Yes, that looks correct. I tested locally to confirm whether npm will raise an exception resolving peer deps, and it unfortunately does. I think we need to add this for everything to install correctly